So 54 supabase integration

jvniorrr opened the pull request:
![Sad Times 😔](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZGI2eXcxZTh2N2thejQ0Y3FsZm5rYnVzY3AyZ2M5MWFzengwM2h6eSZlcD12MV9naWZzX3RyZW5kaW5nJmN0PWc/9miJXdLW3lutj1KM7P/giphy.gif)

- Authentication implementation
- Supabase integration
- Still working on SQL schemas, just need to export them once I am done.

jvniorrr committed:
SO-54 Supabase Integration

jvniorrr committed:
SO-57 REST API Route(s) for Auth, Loader Page

- Temporary Loader page layout
- Dropdown for language selection fixes.
- Cleaned code a bit

jvniorrr committed:
SO-59 Database Schema's

- Supabase reference of the tables and their schemas.
- Enum Values provided as well

jvniorrr committed:
SO-69 Organization page

- Added page view for organizations
- Added check constraints allowing a user to only see their organization ties
- Dashboard admin implemention only admins may see.

hashemJaber reviewed (commented):
Greate job, clean code, reusable, easy to read, and thoughtful, good job

jvniorrr committed:
Invitation by search implementation

- Search for members and invite them to your organization
- Dashboard access given to admin and supervisor roles
- Organization choice to allow for a later SaaS service.
- Import naming fixes for build process
